nengo/neurons.py

```diff
@@ -225,12 +225,16 @@ class LIF(LIFRate):
 
     probeable = ['spikes', 'voltage', 'refractory_time']
 
+    def __init__(self, min_voltage=0, **kwargs):
```

Comment:
This changes the order of parameters for LIF. Are we sure that no-one is doing `LIF(0.03, 0.003)` without the keywords? If they are, is it their own damn fault if this breaks their code?